README.md

```diff
@@ -2,6 +2,49 @@
 ETHOnline 2022 entry by ChainHackers
 https://gamejutsu.app/
 
+# Run locally
+```shell
+yarn install
+```
+add the file ```.env.local``` to the root of the project and in it declare the variable ```NEXT_PUBLIC_ALCHEMY_API_KEY with the key from ALCHEMY_API```
+
+```shell
+yarn generate
+yarn dev
+```
+
+### Development
+```shell
+yarn install
```

Comment:
Is this section different from "Run locally"?

Reply:
I don't think so. deleted